code/modules/food_and_drinks/food/foods/pizza.dm

```diff
@@ -475,6 +477,8 @@
 			playsound(src, 'sound/voice/pizza_time.ogg', 50, FALSE) ///Sound effect made by BlackDog
 		message_admins("[key_name_admin(usr)] has triggered a pizza bomb armed by [key_name_admin(armer)] at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[loc.x];Y=[loc.y];Z=[loc.z]'>(JMP)</a>.")
```

Comment:
```suggestion
```
This should be caught by the `add_attack_logs` proc. IIRC this is also what we've done for the welding tank explosions. But an admin can correct me on this.
Same goes for the one above

Reply:
I just looked at grenade code for its similarities.